README.md

```diff
@@ -1,154 +1,139 @@
-# Nerves.Runtime
+# Nerves Runtime
 
 [![Build Status](https://travis-ci.org/nerves-project/nerves_runtime.svg)](https://travis-ci.org/nerves-project/nerves_runtime.svg)
 [![Hex version](https://img.shields.io/hexpm/v/nerves_runtime.svg "Hex version")](https://hex.pm/packages/nerves_runtime)
 
-Nerves.Runtime is a core component of Nerves. It contains applications and
-libraries that are expected to be useful on all Nerves devices.  Here are some
-of its features:
+Nerves Runtime is a core component of Nerves.
+It contains applications and libraries that are expected to be useful on all Nerves devices.
+Here are some of its features:
 
-* Generic system and filesystem initialization (suitable for use with
-  [bootloader](https://github.com/nerves-project/bootloader))
+* Generic system and filesystem initialization (suitable for use with [bootloader](https://github.com/nerves-project/bootloader))
 * Introspection of Nerves system firmware and deployment metadata
-* A custom shell for debugging and running commands in a `bash` shell like
-  environment
+* A custom shell for debugging and running commands in a `bash`-like environment
 * Device reboot and shutdown
-* A small Linux kernel `uevent` application for capturing hardware change events
-  and more
+* A small Linux kernel `uevent` application for capturing hardware change events and more
 * More to come...
 
-The following sections describe the features in more detail. For even more
-information, consult the [hex docs](https://hexdocs.pm/nerves_runtime).
+The following sections describe the features in more detail.
+For even more information, consult the [hex docs](https://hexdocs.pm/nerves_runtime).
 
-## System initialization
+## System Initialization
```

Comment:
Sigh. I just noticed that most of the Nerves docs used title case and the same with the Elixir docs. However, there's some sentence case mixed into the Elixir docs. I suppose those are mistakes. Anyway, my beef with title case for these docs are the rules and how easy people mess them up, the more formal appearance, and the ambiguity with what's a proper noun in a title, but I guess that I'll deal. Sigh.